python/tvm/micro/model_library_format.py

```diff
@@ -519,7 +570,7 @@ def _export_operator_model_library_format(mod: build_module.OperatorModule, temp
     """
     targets = []
     for target in mod.ir_module_by_target.keys():
-        if str(target.kind) not in ("llvm", "c"):
+        if str(target.kind) not in ("llvm", "c", "gemmini"):
```

Comment:
This seems like an unrelated change?

Reply:
Yeah, sorry for that, will change it back.

Reply:
added a new commit fixing this